casepro/cases/models.py

```diff
@@ -332,11 +332,12 @@ def reopen(self, user, note=None, update_contact=True):
         self.notify_watchers(action=action)
 
     @case_action()
-    def reassign(self, user, partner, note=None):
+    def reassign(self, user, partner, user_assignee=None, note=None):
```

Comment:
Resolved in https://github.com/praekelt/casepro/pull/69/commits/cca4c27f7851039abc13db1dfc3c12af6a79a105